src/StringValue.php

```diff
@@ -148,7 +148,7 @@ public function positionLast($needle): ?int;
 
     /**
      * @param string|StringValue $pattern
-     * @return ArrayValue<array<int|string, string>>
+     * @return ArrayValue<array<string>>
```

Comment:
It can be `array<int|string, string>` when there's named subpattern
`(?P<name>lorem)` => `[0 => 'lorem', 'name' => 'lorem', 1 => 'lorem']`

Reply:
I think I has similar problem on my branch and resolved it [like this](https://github.com/gowork/values/pull/50/commits/7a3290c12430393351dbf2280c12e2f0c6beedf6) :smirk: